resources/definitions/ultimaker_method_base.def.json

```diff
@@ -342,6 +342,11 @@
         "multiple_mesh_overlap": { "value": 0 },
         "optimize_wall_printing_order": { "value": true },
         "prime_blob_enable": { "enabled": false },
+        "prime_tower_enable":
+        {
+            "default_value": "extruders_enabled_count > 1",
+            "enabled": "extruders_enabled_count > 1"
+        },
```

Comment:
This should not be necessary. This setting is already disabled based on the same criterion, and also has a proper resolve function. Do you have cases where it still makes a prime tower with only 1 extruder ?

Reply:
I was getting no prime tower for Method X PLA + PLA, but maybe there was an issue with my Cura resources files.

Reply:
I removed the enable and set the default value to true. This has the desired behavior (Method X PLA + PLA shows a prime tower)